### PR TITLE
add COLA sign-out link

### PR DESCRIPTION
**Pull request by harley-harris**
### JIRA link

[NTRNL-512](https://technologyprogramme.atlassian.net/jira/software/projects/NTRNL/boards/312?selectedIssue=NTRNL-512)

### Description

- Add COLA sign-out link. This link will remove the COLA cookie and redirect to the COLA login page. Users can then sign in again to access the service.

### Work checklist

- [ ] Tests added where applicable
- [ ] No vulnerability added

[NTRNL-512]: https://technologyprogramme.atlassian.net/browse/NTRNL-512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/views/include/sign-out.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <li class="govuk-list govuk-!-margin-top-2">
       <p class="govuk-body">
         {{ userEmailAuth | default(userEmail) }} | 
-        <a class="govuk-link govuk-link--no-underline govuk-link--no-visited-state" href={{ ROUTES.SIGN_OUT_URL }}>Sign out</a>
+        <a class="govuk-link govuk-link--no-underline govuk-link--no-visited-state" href="https://cola.service.cabinetoffice.gov.uk/v2/github-requests/?logout=true">Sign out</a>
```
Comment (Mouhajer-CO) [inline src/views/include/sign-out.html:6]: I would refactor the following environment variable, AUTH_SIGN_IN_URL, as follows:

```js
AUTH_PATH = "https://cola.service.cabinetoffice.gov.uk/v2/github-requests"
AUTH_SIGN_IN_URL = "${AUTH_PATH}/login"
AUTH_SIGN_OUT_URL = "${AUTH_PATH}/?logout=true"
```
and used AUTH_SIGN_OUT_URL on the `href`

File: src/config/index.ts [hunk 2]
```diff
@@ -82,6 +82,11 @@ export const FEATURE_FLAG_ENABLE_AUTH = getEnvironmentValue('FEATURE_FLAG_ENABLE
 export const FEATURE_FLAG_ENABLE_NOTIFY = getEnvironmentValue('FEATURE_FLAG_ENABLE_NOTIFY', 'false');
 export const FEATURE_FLAG_ENABLE_DYNAMO = getEnvironmentValue('FEATURE_FLAG_ENABLE_DYNAMO', 'false');
 
+// Auth URLs
+export const AUTH_PATH = 'https://cola.service.cabinetoffice.gov.uk/v2/github-requests';
```
Comment (Mouhajer-CO) [inline src/config/index.ts:86]: The `AUTH_PATH` should be dynamic and fetched from the `envs`, we use to have it on our `env` named AUTH_SIGN_IN_URL as you can see [here](https://github.com/cabinetoffice/github-requests-app/blob/56a9fff70b7f4ede51acaa82dfddc0fa54a4fd5f/README.md?plain=1#L31)

Comment (harley-harris) [reply src/config/index.ts:86]: Yep you're right, forgot to import it

**Review by Mouhajer-CO — APPROVED** [review]
🔥

**Pull request merged**